tensorflow/python/tools/saved_model_cli_test.py

```diff
@@ -31,12 +31,16 @@
 from tensorflow.core.framework import types_pb2
 from tensorflow.core.protobuf import meta_graph_pb2
 from tensorflow.python.debug.wrappers import local_cli_wrapper
+from tensorflow.python.eager import def_function
+from tensorflow.python.framework import constant_op
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import tensor_spec
 from tensorflow.python.platform import test
-from tensorflow.python.tools import saved_model_cli
-
+from tensorflow.python.saved_model import save
+import saved_model_cli
```

Comment:
why?

Reply:
Oops, was using it to test locally. I've not built tensorflow locally, so bazel test will take a lot of time for first time compilation. correcting it.